### ISSUE TITLE
Custom pin reverts to native map pin/marker

### ISSUE THREAD

**Issue by pcitachi007** [TorbenK/TK.CustomMap#385]
Hi there my code currently loads user's location, then loads spots around them. These spots are some I hold in my local DB. However when I leave the move around the map and go back to an already loaded location, if the location is a "recognized place" (i.e. Vons, Albertsons) on the map the pin changes to this: 
![image](https://user-images.githubusercontent.com/86023965/122593823-0e725380-d01b-11eb-92b5-1713920626cf.png).  
This seems to be the built in marked location pin on iOS. Is there something I am missing or needs to be handled with custom rendering?

**Comment by pcitachi007** [comment]
I see this is an ongoing issue when IsClusteringEnabled... I am following this thread but it appears no updates have come from it...https://github.com/TorbenK/TK.CustomMap/issues/292

**Comment by pcitachi007** [comment]
Not sure if anyone cares anymore for this nugget, but I ended up figuring this out on my own by doing some digging in the backend. This was one of the few Xamarin Maps plugin that worked for the most part and since my apps were still on Xamarin I figured I would put in some work. 
For iOS, the issues lies within the `GetViewFromAnnotation `referencing a `WrapperAnnotation `type instead of `CKCluster`. For Android the issue was within the `OnBeforeClusterItemRendered`.. essentially putting that whole block in the `OnClusterItemRendered `and it was fixed. Also, I believe this was always an issue, but I noticed in the sample project pins are added via a quick block and then to test the the `EnabledClustering `function, when it was set to true, then false, the pins would remain normal pins. The issue was when a pin was already loaded and the mapregion moved and then moved back the pins would change to default iOS or Google pins.. I'm not going to get into the coding on here, but if anyone is interested just comment or message me and I can put my code.
